mentions/src/main/kotlin/io/spine/examples/pingh/mentions/GitHubClientProcess.kt

```diff
@@ -134,12 +146,30 @@ public class GitHubClientProcess :
             .toSet()
 
     /**
-     * Supplies this instance of the process with a service allowing to access GitHub.
+     * Removes from the set mentions that are already saved in the [UserMentions] projection.
+     */
+    private fun Set<UserMentioned>.onlyNew(context: EventContext): Set<UserMentioned> {
```

Comment:
I don't understand this approach at all. You are in a `ProjectionRepository`. So theoretically, you should be able to route or not route (return an empty `Set` of IDs) any incoming event telling someone was mentioned. 

Why do you still have to read something in this weird way?

Let's discuss vocally.

In general, this approach with `Reader`s should be your last resort. In most cases, own state of an entity should be enough. Most common scenario which assumes accessing read-side like this is handing a signal in a process manager. And your scenario is not it.